test.sh

```diff
@@ -77,7 +77,13 @@ function dcdown {
     # Shutting down all containers associated with this project
     docker-compose -f $COMPOSE_FILE_LOC \
                    -p $COMPOSE_PROJECT_NAME \
-               down
+                    down 
+}
+
+function static_tester_up {
```

Comment:
while I'm happy that the same travis build runs the frontend tests, I'd prefer to have a different `frontend-test.sh` to test the frontend stuff. (or a way to specify that we only want to run the frontend tests in `test.sh`, maybe something like ./`test.sh --js-only`). I'll let you choose whatever seems more convenient to you.

This gives us the benefit of not having to set up the database and other random services, just to run the js tests which will never touch those services.